Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM centos/httpd:latest
 MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq-appliance-build
 
+## Systemd
+ENV container oci
```

Comment:
@abellotti It should not , the variable is a hint to systemd that it is running inside a container.